streaming/src/main/scala/org/apache/gearpump/streaming/DAG.scala

```diff
@@ -21,30 +21,32 @@ package org.apache.gearpump.streaming
 import org.apache.gearpump.partitioner.{PartitionerObject, PartitionerDescription, Partitioner}
 import org.apache.gearpump.streaming.task.TaskId
 import org.apache.gearpump.util.Graph
-import upickle._
 
-import scala.collection.JavaConversions._
-
-
-case class DAG(processors : Map[ProcessorId, ProcessorDescription], graph : Graph[ProcessorId, PartitionerDescription]) extends Serializable {
+case class DAG(version: Int, processors : Map[ProcessorId, ProcessorDescription], graph : Graph[ProcessorId, PartitionerDescription]) extends Serializable {
```

Comment:
Does it mean that for each processor, it can only has one partitioner for outgoing edges?